CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+We'd love to accept your contributions to this project! There are just a few guidelines you need to follow.
+
+## Bugs
+
+Bug reports should be opened up as [issues](https://help.github.com/en/github/managing-your-work-on-github/about-issues) on the [go-vela/community](https://github.com/go-vela/community) repository!
+
+## Feature Requests
+
+Feature Requests should be opened up as [issues](https://help.github.com/en/github/managing-your-work-on-github/about-issues) on the [go-vela/community](https://github.com/go-vela/community) repository!
+
+## Pull Requests
+
+**NOTE: We recommend you start by opening a new issue describing the bug or feature you're intending to fix. Even if you think it's relatively minor, it's helpful to know what people are working on.**
+
+We are always welcome to new PRs!
```

Comment:
We are always open to new PRs, or We always welcome new PRs

Reply:
@KellyMerrick sure I can make the change 👍 

That same statement exists in all our contributing docs today so I'll do my best to propagate that change across all the repos.